docs/openapi.yaml

```diff
@@ -4793,9 +4789,29 @@ components:
               type: string
     UpdateFileRequest:
       type: object
+      required:
+        - type
+        - identifier
       properties:
+        type:
+          type: string
+          enum:
+            - UpdateFileRequest
         identifier:
           type: string
+        license:
+          type: string
```

Comment:
There is a restriction here, which could be an enum, but we need the enum (and to keep it updated)